Automattic/sensei
# Fix draft questions blocks focus & hiding

### Changes proposed in this Pull Request

* Fix quiz block grabbing focus for new lessons
* Fix all draft questions being hidden when quiz block is not focused
* Fix newly inserted quiz block losing focus

### Testing instructions

* Create a new lesson. 
  - Focus should be on the lesson title.
  - Quiz block should have one draft question, always visible.
* Add a **new quiz block** to a lesson (remove it first if there is already one). 
  - First question title should receive focus.
* Add questions, click outside quiz block.
  - Only the last auto-inserted draft should be hidden 


### Screenshot / Video

<img width="787" alt="image" src="https://user-images.githubusercontent.com/176949/111652240-37f6c580-8807-11eb-84c9-6fae8fad834a.png">
